pages/initiatives/gsoc/gsoc2023ideas.md

```diff
@@ -7,7 +7,7 @@ permalink: /initiatives/gsoc/gsoc2023ideas
 
 # {{page.title}}
 
-[ZAP](#owaspzap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt) | [Maryam](#owaspmaryam) | [SecureTea](#project-securetea) | [PyGoat](#project-pygoat) | [RiskAssessmentFramework](#risk-assessment) | [Juice Shop](#owaspjuiceshop)
+[ZAP](#owasp-zap) | [Bug Logging Tool (BLT)](#bug-logging-tool-blt) | [Maryam](#owasp-maryam) | [SecureTea](#owasp-securetea) | [PyGoat](#owasp-pygoat) | [RiskAssessmentFramework](#owasp-risk-assessment-framework) | [Juice Shop](#owasp-juice-shop) | [OWASP WrongSecrets](#owasp-wrongsecrets)
```

Comment:
Amazingly we did actually check this earlier. It was working without hyphen (though maybe only on github not when deloyed :shrug:  ... with I could remember if we tried both :cry:)

Reply:
I'm getting some inconsistent behaviour as well. Curious